### ISSUE TITLE
Duplicate id on div.input-group child input

### ISSUE THREAD

**Issue by chris246** [FlipperPA/django-tempus-dominus#39]
Hi,

the Django Widget generates the following incorrect HTML Code:
```html
<div class="input-group " id="id_myinput" data-target-input="nearest">
    <input type="text" name="myinput" class="form-control datetimepicker-input" required="True" id="id_myinput" data-toggle="datetimepicker" data-target="#id_myinput">
</div>
```
The Element `#id_myinput` appears twice in DOM, breaking e.g. labels focus functionality.

Version 5.1.2.1

**Comment by chris246** [comment]
Hi, thanks for your fast fix. This version works.

Also, your approach is similar what I would have done, but I was not sure whether this would break scripts or stuff 😄

**Comment by FlipperPA** [comment]
@chris246 Thanks again for the report. Sorry for the delay, but I've merged this and pushed out version 5.1.2.2:

https://pypi.org/project/django-tempus-dominus/

Enjoy!

**Comment by chris246** [comment]
Thanks.
No problem. It was not that critical and your initial fix also did the job.
